package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "react-native-inview",
   "description": "Detects if a ScrollView or ListView element is inside the users viewport and reports back on changes.",
+  "version": "0.0.2",
```

Comment:
Dont add version, i use "semantic release" that will automate the releases, versions, and tags.

Reply:
My bad, that was meant for something else 🙈